cupy/cuda/cub.pyx

```diff
@@ -57,18 +57,32 @@ cdef extern from 'cupy_cub.h':
 # Python interface
 ###############################################################################
 
-cpdef _preprocess_array(ndarray arr, axis, bint keepdims, str order):
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+cpdef bint _contig_axes(tuple axes):
```

Comment:
I wonder if this function can be put to somewhere else, like `cupy.core.internal` or `cupy.util`? This way we can reuse it for cuFFT later, which follows a similar logic to get contiguous axes.

Reply:
Yeah, I was intending to modify cupy.fft to use this and avoid some other minor overhead in a separate follow-up PR

Reply:
I have not addressed this comment yet. `cupy.core.internal` seems like an okay place to me.

Reply:
I missed addressing this one yesterday when I, but have now moved this as suggested.